src/components/paragraph/audience_topics/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { v4 as uuidv4 } from 'uuid'
+import Link from 'next/link'
+import { getTagsList } from '@/utils/utils'
+
+const AudienceTopics = ({ paragraph }) => {
+  if (!paragraph) paragraph = []
+  // Build tag list
+  const tags = getTagsList(paragraph)
+  // only render tags where length is greater than 0
+  if (tags.length === 0) return null
+
+  const tagsList = tags.map((tag) => (
+    <div key={uuidv4()}>
+      <div className="vads-u-margin-right--1 vads-u-margin-bottom--1 medium-screen:vads-u-margin-bottom--0">
+        <Link href={`${tag.path.alias}/${tag.name}`}>
+          <a className="vads-u-margin-bottom--1p5 usa-button-secondary vads-u-font-size--sm vads-u-border--1px vads-u-border-color--primary vads-u-padding--0p25 vads-u-padding-x--0p5 vads-u-margin-left--1p5 vads-u-text-decoration--none vads-u-color--base">
+            {tag.name}
+          </a>
+        </Link>
```

Comment:
I noticed in a record event call to an analytics click event that we currently do not support on next-build so I left it out to prevent errors.

content-build version
```
<a href="{{ entity.entityUrl.path }}"
class="vads-u-margin-bottom--1p5 usa-button-secondary vads-u-font-size--sm vads-u-border--1px vads-u-border-color--primary vads-u-padding--0p25 vads-u-padding-x--0p5 vads-u-margin-left--1p5 vads-u-text-decoration--none vads-u-color--base"
style="border-radius: 3px; line-height: 1.3;"
onclick="recordEvent({ 'event': 'nav-page-tag-click', 'page-tag-click-label': '{{ entity.name }}', 'page-tag-category-label': '{{  entity.categoryLabel  }}' });">
{{ entity.name }}
</a>
            ```